requirements.txt

```diff
@@ -1,6 +1,7 @@
 # install using: pip3 install --upgrade -r requirements.txt
 git+https://github.com/owasp-sbot/OSBot-Utils.git
-git+https://github.com/owasp-sbot/OSBot-AWS.git
+git+https://github.com/k8-proxy/OSBot-AWS.git
+git+https://github.com/filetrust/GW-Bot.git
 boto3
```

Comment:
we shouldn't be using repos outsite the owasp-sbot

also this repo should not have any dependency on https://github.com/filetrust/GW-Bot.git (we will need to bring some of that code in here)

I'm going to edit this file so that I can merge your other changes)

Reply:
@dinis-cruz-gw : Sorry, I forgot to fix this file.